substrate/frame/transaction-payment/asset-conversion-tx-payment/src/benchmarking.rs

```diff
@@ -0,0 +1,140 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Benchmarks for Asset Conversion Tx Payment Pallet's transaction extension
+
+use super::*;
+use crate::Pallet;
+use frame_benchmarking::v2::*;
+use frame_support::{
+	dispatch::{DispatchInfo, PostDispatchInfo},
+	pallet_prelude::*,
+	traits::fungibles::Inspect,
+};
+use frame_system::RawOrigin;
+use sp_runtime::traits::{AsSystemOriginSigner, DispatchTransaction, Dispatchable};
+
+pub trait ExtConfig: Config {
```

Comment:
```suggestion
pub trait BenchConfig: Config {
```
Or something similar is more expressive IMHO.

Reply:
Renamed.